src/main/resources/db/migration/common/V2025.08.01.0002__Prisoner_overview_3.sql

```diff
@@ -0,0 +1,139 @@
+DROP VIEW prisoner_overview;
+
+CREATE OR REPLACE VIEW prisoner_overview AS
+WITH all_prisoners AS (
+    SELECT prison_number FROM aln_assessment
+    UNION
+    SELECT prison_number FROM ldd_assessment
+    UNION
+    SELECT prison_number FROM education
+    UNION
+    SELECT prison_number FROM condition
+    UNION
+    SELECT prison_number FROM challenge
+    UNION
+    SELECT prison_number FROM strength
+    UNION
+    SELECT prison_number FROM plan_creation_schedule
+    UNION
+    SELECT prison_number FROM review_schedule
+    UNION
+    SELECT prison_number FROM elsp_plan
+)
+SELECT
+    ap.prison_number,
+
+    -- Latest ALN assessment
+    aln.has_need AS has_aln_need,
+
+    -- Latest LDD assessment
+    ldd.has_need AS has_ldd_need,
+
+    -- Latest education record
+    edu.in_education AS in_education,
+
+    -- Any active condition
+    (cond.exists IS NOT NULL) AS has_condition,
+
+    -- Any active non-screener challenge
+    (chal.exists IS NOT NULL) AS has_non_screener_challenge,
+
+    -- Any active non-screener strength
+    (str.exists IS NOT NULL) AS has_non_screener_strength,
+
+    -- Plan creation deadline
+    pcs.deadline_date AS plan_creation_deadline_date,
+
+    -- Latest scheduled review deadline
+    rsched.deadline_date AS review_deadline_date,
+
+    -- Combined deadline (review > plan creation > null)
+    COALESCE(rsched.deadline_date, pcs.deadline_date) AS deadline_date,
+
+    -- Has a plan in elsp_plan
+    (plan.exists IS NOT NULL) AS has_plan,
+
+    -- Plan declined if status is EXEMPT_PRISONER_NOT_COMPLY
+    (pcs.status = 'EXEMPT_PRISONER_NOT_COMPLY') AS plan_declined,
```

Comment:
this is the actual change - I'll set the status to plan declined when I see this as true.